src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -116,6 +121,15 @@
               optional = true)
     private Boolean overwriteExistingWorkspace = false;
 
+    @Argument(fullName = BATCHSIZE_ARG_NAME,
+              shortName = BATCHSIZE_ARG_NAME,
+              doc = "Batch size to group samples in during import. " +
+                    "This will minimize memory consumption, but might take longer to complete." +
+                    "Also, if batching creates too many fragments in GenomicsDB, run consolidation" +
+                    " to improve feature read time. Defaults to " + DEFAULT_ZERO_BATCH_SIZE,
+              optional = true)
+    private int batchSize = DEFAULT_ZERO_BATCH_SIZE;
```

Comment:
The tool will not blow up. The batch size is used to calculate lower and upper bound of sample indexes. If batch size is larger than number of samples, last sample index will be used. I will add a test with this.